samples/dashboards/demos/mathmodifier-formula/demo.js

```diff
@@ -0,0 +1,154 @@
+const container1 = document.querySelector('#container');
+const container2 = document.querySelector('#container2');
+const cursor = new Dashboards.DataCursor();
+const MathModifier = Dashboards.DataModifier.types.Math;
```

Comment:
I don't think we want to show these methods and Classes, do we? When I said we need a public demo for MathModifier, I meant MathModifier in Dashboards/DataGrid, not a copy of the demo from internal data-tools.

Reply:
Then I suggest to extend one of the existing Dashboards/DataGrid demos. Or do you like to have a new Dashboards demo?